### DISCUSSION TITLE
How do I get the Galaxy Brain badge on my profile ?

### DISCUSSION THREAD

**Discussion by jeesNeetable** [community/community#58864] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I get the Galaxy Brain badge on my profile, any one knows

**Comment by jees47** [comment, marked as answer]
Read this link https://github.com/orgs/community/discussions/18293 it should cover all your questions

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
